### ISSUE TITLE
Codecombat cloudrip mountain ( Crag Tag ) bug

### ISSUE THREAD

**Issue by Elvis-Tam** [codecombat/codecombat#5787]
There is currently a bug in cloudrip mountain's first level : Crag Tag. It is telling me to use my boots, but it also tell me that my boots is restricted. I can't choose the boots and I can't start the level. I found this bug about 3 weeks ago, and now I still can't select the boots. Someone in codecombat discourse said that he used winged boots to enter the level, but I don't have enough money. https://discourse.codecombat.com/t/cloudrip-mountain-first-level-bug/20332 ( the issue that I created for this bug )

**Comment by envwarby** [comment]
I just made an account here to post this exact same bug glad to see somebody is already on it I guess all I have to add here is a screenshot:
![RestrictedMandatoryEquip](https://user-images.githubusercontent.com/68952187/88795571-8d129800-d1a0-11ea-9fc2-277a46d67439.jpg)

**Comment by envwarby** [comment]
Some additional info not sure if this is helpful:

I went to another level that has no restrictions  and tried a bunch of equipment permutations:
-I tried entering the level with "No Boots"
-I tried entering the level with "Leather Boots"
-I tried entering the level with "Simple Boots"

no success

**Comment by envwarby** [comment]
In one of those discourse threads somebody claimed that having the "Boots of Leaping" equipped will get you past this bug I can confirm this to be TRUE! It costs 741 gems. I did not have enough money but there is a challenge level on the first map ( that looks like a dungeon) in the bottom right corner which you can play over and over again ( with increased difficulty) 

 was able to grind enough gems to buy the boots.

 I hope this helps future players finding this thread.

**Comment by Elvis-Tam** [comment]
yes, I hope so too, anyways I solved it a long time ago since I check out
the shop and it said the boots of leaping got a jump code

**Comment by derekclee** [comment]
My son ran into this bug and I can confirm that equipping the `Boots of Leaping` got us past this bug.
